src/HotChocolate/ApolloFederation/src/ApolloFederation/Extensions/RequiresDirectiveTypeDescriptorExtensions.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using HotChocolate.Language;
+using HotChocolate.Types;
+
+namespace HotChocolate.ApolloFederation.Extensions
```

Comment:
This should be moved into ApolloFederationFieldDescriptorExtensions.

Reply:
Moved to ApolloFederationFieldDescriptorExtensions